File: src/primus/pawel/Exercise_075.java
// Don't give me five!
// 7kyu
// https://www.codewars.com/kata/5813d19765d81c592200001a/train/java
// 2021-04-13 T:20:00:41
package primus.pawel;

class Exercise_075 {
    public static int dontGiveMeFive(int start, int end) {
        int count = 0;
        for (int i = start; i <= end; i++) {
            if (!String.valueOf(i).contains("5")) count++;
        }
        return count;
    }

    public static void main(String[] args) {
        dontGiveMeFive(4, 26);
    }

}
// return IntStream.rangeClosed(start, end).filter(x -> !String.valueOf(x).matches(".*5.*")).count();

File: src/JS/Exercise_075.js

// Remove duplicates from list
// 8 kyu
// https://www.codewars.com/kata/57a5b0dfcf1fa526bb000118/train/javascript
// 2022-05-28 T:12:28:46

function distinct(a) {
    return [...new Set(a)];
}